### ISSUE TITLE
Convertion rate 

### ISSUE THREAD

**Issue by RubioB** [FelixKrueger/Bismark#357]
Hi Felix, 

I carried out the verifications of the convertion rates using the phage lambda and the chloroplast genome. With lambda phage I have a convertion rate close to 100% however with the chloroplastic genome I am at 98.8 %.
What does this mean for the rest of my analyzes ? Can I still use my data ? How can/should I take this into account ? 

Thanks in advance !

Bernadette

**Comment by FelixKrueger** [comment]
I personally would just acknowledge these rates, keep them in mind - and move on. I suppose it could mean that in 1 out of 100-1000 unmethylated positions you might observe one position methylated as a consequence of non-conversion rather than being actually methylated.

For real genomic positions (as opposed to an unmethylated spike-in control) you do however also have true CpG and non-CG methylation that are indiscriminable from non-conversion events. And furthermore, there are some positions that appear to be chemically protected from conversion, see e.g. here:  [Evidence Suggesting Absence of Mitochondrial DNA Methylation](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5671948/).

So bottom line: your level of conversion looks fine, but if all your major changes are in the region of up to 1% methylation diffference you need to be aware that this is potentially also the region of bisulfite non-conversion error.s
